templates/artist/artist_detail.html

```diff
@@ -184,11 +206,15 @@ <h1 class="header-font">{{ artist.name }}</h1>
             <hr>
         </div>
         <!-- Updates -->
-        <h2>Updates</h2>
-        <ul>
-            {% for update in updates %}
-                <li>{{ update.created_datetime }}: {{ update.text }}</li>
-            {% endfor %}
+        <ul class="vertical menu" data-accordion-menu>
+            <li>
+            <a href= "#"><h2>UPDATES</h2></a>
+            <ul class="menu vertical nested is-active">
+                {% for update in updates %}
+                    <li>{{ update.created_datetime }}: {{ update.text }}</li>
+                {% endfor %}
+            </li>
+            </ul>
```

Comment:
That's how it said to do it in the foundation documentation. Maybe I misread it somehow

http://foundation.zurb.com/sites/docs/dropdown-menu.html